NitroxClient/Communication/Packets/Processors/PickupItemProcessor.cs

```diff
@@ -8,13 +10,16 @@ namespace NitroxClient.Communication.Packets.Processors
 {
     public class PickupItemProcessor : ClientPacketProcessor<PickupItem>
     {
+        
+
         public override void Process(PickupItem pickup)
         {
             Optional<GameObject> opGameObject = GuidHelper.GetObjectFrom(pickup.Guid);
 
             if (opGameObject.IsPresent())
             {
                 UnityEngine.Object.Destroy(opGameObject.Get());
+                NitroxServiceLocator.LocateService<Entities>().RemoveEntity(pickup.Guid);
```

Comment:
Should we inject this into the processor through the constructor instead of using the service locator?